subsys/net/lib/azure_iot_hub/Kconfig

```
@@ -25,16 +65,17 @@ config AZURE_IOT_HUB_SEND_TIMEOUT
 	  function will not block indefinitely. To configure the length of the timeout,
 	  use AZURE_IOT_HUB_SEND_TIMEOUT_SEC.
 
-if AZURE_IOT_HUB_SEND_TIMEOUT
-
-config AZURE_IOT_HUB_SEND_TIMEOUT_SEC
-	int "Send timeout"
-	default 60
+config AZURE_IOT_HUB_STATIC_IP_ADDRESS
+	string "Static IP address"
 	help
-	  Timeout in seconds to use when the Azure IoT Hub socket is configured to
-	  send with a timeout by enabling AZURE_IOT_HUB_SEND_TIMEOUT.
+	  Set a static IP address to use when connecting to Azure IoT Hub.
+	  Leave the string empty to use DNS to resolve the IoT Hub hostname instead.
 
-endif
+config AZURE_IOT_HUB_NATIVE_TLS
+	bool "Enable native TLS socket"
```

Please remove `Enable` as per guide lines:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

```suggestion
	bool "Native TLS socket"
```